source/localizable/feedspec/index.el.html.md.erb

```diff
@@ -171,6 +171,7 @@ ImageBot v1".
 [Additional Image Link](#additional-image-link)        | String  | <span class="mono">400</span>  | Όχι<b><sup>1</sup></b>| `https://www.someurl.com/images/322/front_view.jpg`
 [Category Name](#category-name)                        | String  | <span class="mono">250</span>  | Ναι         | `Κινητά τηλέφωνα`
 [Price](#price)                                        | Decimal | <span class="mono">- (έως 2 δεκαδικά ψηφία)</span>    | Ναι         | `23.22`
+[Vat Rate](#vat-rate)                                  | Decimal | <span class="mono">-</span>    | Όχι         | `24.00`
```

Comment:
we can also point out that this field accepts precision 2 (same as `price` above)


```suggestion
[Vat Rate](#vat-rate)                                  | Decimal | <span class="mono">- (έως 2 δεκαδικά ψηφία)</span> | Όχι         | `24.00`
```

Reply:
Done